src/definitions.d.ts

```diff
@@ -67,6 +67,11 @@ export interface Theme {
     lightColorScheme: string;
     darkColorScheme: string;
     immediateModify: boolean;
+    cssVariableBg: string;
+    cssVariableText: string;
+    cssVariableScrollBar: string;
+    cssVariableSelection: string;
```

Comment:
Could you please answer this?

Reply:
To be honest at the time I kind of forgot that darkreader could have a light theme. I have changed it now so that there's background/text values for both light and dark. I've left scrollbar and selection as global because I think there corresponding color values are also global.

Reply:
So if I used a theme for github.com which uses custom scrollbar variable but on my default theme uses another custom scrollbar value. How would this code handle that case? It seems that this code would only handle the one that was set last.

Reply:
Fair point. I've just tested, having custom themes works to an extent as darkreader still stores the color associated with the css variable. However, if the pywal theme is changed the colors for the custom theme do not. I'll try figuring it out.

Reply:
I think i've fixed the issue. It wasn't due to the variables but where the cssUpdate function had been placed and called. It's works with presets, site specific and the all-site themes.